governance/polls/Ratification Poll for Growth Core Unit Budget, GRO-001 (MIP40c3-SP37) - October 11, 2021.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ratification Poll for Growth Core Unit Budget, GRO-001 (MIP40c3-SP37) - October 11, 2021
+summary: MIP40c3-SP37 adds the budget for Core Unit GRO-001: Growth for November 2021 through June 2022.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp37-growth-core-unit-budget-gro-001/10228
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-10-11T16:00:00
+end_date: 2021-10-25T16:00:00
+---
+# Ratification Poll for Growth Core Unit Budget, GRO-001 (MIP40c3-SP37) - October 11, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, October 11 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP37 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This budget proposal covers a total of 8 months (November and December 2021 and H1 2022) to cover the remaining months until the end of the year and the first six months of 2022.
+
+Growth CU is requesting an eight months budget allocation of 2,208,484 DAI. 1,117,663 DAI will cover operations, and 1,100,000 DAI will cover the work we’ll do with our partners in the latest stages of the Growth Cycle (adoption campaigns, educational content, etc.). The remaining budget from the previous months (posted in our weekly updates at the beginning of each month) is deducted from the total of the current budget.*
```

Comment:
Formatting of the quotation here isn't working correctly
I think you probably need an asterisk at the start and end of each line to fix it